pytorch_finufft/checks.py

```diff
@@ -21,31 +21,31 @@ def check_devices(*tensors: torch.Tensor) -> None:
             )
 
 
-def check_dtypes(values: torch.Tensor, points: torch.Tensor) -> None:
+def check_dtypes(data: torch.Tensor, points: torch.Tensor, name: str) -> None:
```

Comment:
`name` variable for better error message formatting, correct?
Possible to put a default value for it? E.g. "first argument" or "input array"?

Reply:
Correct. It is either `Values` or `Targets` for our purposes. I suppose in type 3 maybe it has a third name